src/lib/ConfigResolver/ConfigurableRegistrationContentTypeLoader.php

```diff
@@ -8,30 +8,35 @@
 
 namespace Ibexa\User\ConfigResolver;
 
-use Symfony\Component\OptionsResolver\OptionsResolver;
+use Ibexa\Contracts\Core\Repository\Repository;
+use Ibexa\Core\MVC\ConfigResolverInterface;
 
 /**
  * Loads the registration content type from a configured, injected content type identifier.
  */
-class ConfigurableRegistrationContentTypeLoader extends ConfigurableSudoRepositoryLoader implements RegistrationContentTypeLoader
+class ConfigurableRegistrationContentTypeLoader implements RegistrationContentTypeLoader
```

Comment:
Any reason why `ConfigurableSudoRepositoryLoader` was dropped here?

Reply:
I was gonna say "because it doesn't need it anymore, but actually it seems that it does...